ofEasyCam error on clean develop checkout

../../../openFrameworks/3d/ofEasyCam.cpp: In member function 'void ofEasyCam::updateMouse()':
../../../openFrameworks/3d/ofEasyCam.cpp:242:59: error: 'FLT_EPSILON' was not declared in this scope
../../../openFrameworks/3d/ofEasyCam.cpp:244:61: error: 'FLT_EPSILON' was not declared in this scope
make: **\* [obj/Debug/openFrameworks/3d/ofEasyCam.o] Error 1
there has been a problem compiling Debug OF library
please report this problem in the forums
chown: cannot access `../../lib/linux64/*': No such file or directory

Seems to be related to the commit 2 hours ago which fixed easycam...?

--- a/libs/openFrameworks/utils/ofConstants.h
+++ b/libs/openFrameworks/utils/ofConstants.h
@@ -312,6 +312,7 @@ typedef ofBaseApp ofSimpleApp;
 #include <iomanip>  //for setprecision
 #include <fstream>
 #include <algorithm>
+#include <cfloat>
 using namespace std;
 
 #ifndef PI

